### PR TITLE
[19432] Clarify super client discovery information

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**

### CODE REVIEW

File: docs/fastdds/discovery/discovery_server.rst [hunk 1]
```diff
@@ -79,9 +79,17 @@ In this architecture there are several key concepts to understand:
...
     addresses are acceptable addresses.
 
-- A |SUPER_CLIENT| is a *client* that receives all the discovery information known by the *server*, in opposition to
+- A |SUPER_CLIENT| is a *client* that receives the discovery information known by the *server*, in opposition to
   *clients*, which only receive the information they need.
 
+  .. note::
+
+    A *Super_client* does not behave as a *Server* as it only receives the discovery information through the *Server* it
+    is connected to.
```
Comment (EduPonz) [inline docs/fastdds/discovery/discovery_server.rst:88]: Mind the line length
```suggestion
    A |SUPER_CLIENT| does not behave as a *Server* as it only receives the discovery information through the *Server* to which it is connected.
```

File: docs/fastdds/discovery/discovery_server.rst [hunk 2]
```diff
@@ -79,9 +79,17 @@ In this architecture there are several key concepts to understand:
...
+    Also, the discovery information the *Server* relays concerns only about the endpoints (DataReaders and DataWriters)
+    that has been discovered in the discovery server network.
```
Comment (EduPonz) [inline docs/fastdds/discovery/discovery_server.rst:90]: The server does relay information about participants, but only if they have endpoints. IDK whether we even need this sentence

File: docs/fastdds/discovery/discovery_server.rst [hunk 3]
```diff
@@ -79,9 +79,17 @@ In this architecture there are several key concepts to understand:
...
+    Any DomainParticipant discovered by the *Server* with no endpoints will not be known by the *Super_client*.
```
Comment (EduPonz) [inline docs/fastdds/discovery/discovery_server.rst:91]:
```suggestion
    Any DomainParticipant discovered by the *Server* with no endpoints will not be known by the |SUPER_CLIENT|.
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

File: docs/fastdds/discovery/discovery_server.rst (hunk 1 above)
Comment (JLBuenoLopez-eProsima) [reply docs/fastdds/discovery/discovery_server.rst:88]: It is just in the line length limit :laughing: The CI has passed without complaining :wink:

File: docs/fastdds/discovery/discovery_server.rst (hunk 2 above)
Comment (JLBuenoLopez-eProsima) [reply docs/fastdds/discovery/discovery_server.rst:90]: I agree. With the first and latest sentence all the required information is already relayed.

**Review by EduPonz — APPROVED** [review]

**Pull request merged**